cl/search/management/commands/cl_index_parent_and_child_docs.py

```diff
@@ -199,19 +199,21 @@ def get_documents_to_update_or_remove(
     child_documents_to_update = []
     documents_to_delete = []
     event_ids = list(events_to_update.values_list("id", flat=True))
-    for event_ids_chunk in chunks(event_ids, chunk_size):
+    event_ids_count = len(event_ids)
+    processed_count = 0
+    for i, event_ids_chunk in enumerate(
+        batched(event_ids, chunk_size), start=1
+    ):
```

Comment:
We're using enumerate in this loop but don't seem to use the variable `i`. Is there a reason to use the enumerate function?

Reply:
Yes, this is now unnecessary. I was using it to compute the progress, but it's simpler to just use the total number of items to be processed and the current progress. Removed. Thanks for pointing it out.